sympy/simplify/tests/test_cse.py

```diff
@@ -67,7 +70,7 @@ def test_cse_single2():
     substs, reduced = cse(e)
     assert substs == [(x0, x + y)]
     assert reduced == [sqrt(x0) + x0**2]
-    assert isinstance(cse(Matrix([[1]]))[1][0], Matrix)
+    assert isinstance(cse(Matrix([[1]]))[1], Matrix)
```

Comment:
This doesn't fit the docstring anymore either:

```
    Returns
    =======

    replacements : list of (Symbol, expression) pairs
        All of the common subexpressions that were replaced. Subexpressions
        earlier in this list might show up in subexpressions later in this
        list.
    reduced_exprs : list of sympy expressions
        The reduced expressions with all of the replacements above.
```
cse has always returned lists of expressions and for the single matrix case, it should return a list containing a single matrix.